hoot-core/src/main/cpp/hoot/core/io/OgrWriter.cpp

```diff
@@ -270,7 +270,8 @@ void OgrWriter::_createLayer(boost::shared_ptr<const Layer> layer)
       {
         options["FEATURE_DATASET"] = layer->getFdName();
         // speed up bulk inserts.
-        options["FGDB_BULK_LOAD"] = "YES";
+        // NOTE: Seems to be depreciated in GDAL 2.0+
+        //options["FGDB_BULK_LOAD"] = "YES";
```

Comment:
This is good.  I've seen warnings before about `FGDB_BULK_LOAD` being deprecated.  Will this change affect the output of some of the unit tests?

Reply:
I didn't see any when I ran glacial. I'll give it another run with test-all.

Reply:
Maybe it was only happening in the regression tests (which don't care about output text only results) or it could be completely ignored.  Jenkins got past that part so it shouldn't be an issue.